packages/pxweb2-ui/src/lib/components/VariableBox/VariableBoxContent/VariableBoxContent.tsx

```diff
@@ -111,15 +117,30 @@ export function VariableBoxContent({
   }, [hasSevenOrMoreValues, hasTwoOrMoreValues, debouncedSearch, values]);
 
   useEffect(() => {
-    if (totalChosenValues === 0) {
+
+    function compareSearchedAndChosenValues (searchedValues: string | any[], chosenValues: string | any[]) {
```

Comment:
Do we need the "any" type here? Or can we use something more specific that has typing on it?